src/test/java/org/assertj/core/internal/classes/Classes_assertIsPackagePrivate_Test.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2019 the original author or authors.
+ */
+package org.assertj.core.internal.classes;
+
+import org.assertj.core.internal.ClassesBaseTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.assertj.core.error.ClassModifierShouldBe.shouldBePackagePrivate;
+import static org.assertj.core.test.TestData.someInfo;
+import static org.assertj.core.util.FailureMessages.actualIsNull;
+
+@DisplayName("Classes assertIsPackagePrivate")
+class Classes_assertIsPackagePrivate_Test extends ClassesBaseTest {
+
+  @Test
+  public void should_fail_if_actual_is_null() {
+    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> classes.assertIsPackagePrivate(someInfo(), null))
+                                                   .withMessage(actualIsNull());
+  }
+
+  @Test
+  public void should_pass_if_actual_is_a_package_private_class() {
+    classes.assertIsPackagePrivate(someInfo(), PackagePrivateClass.class);
+  }
+
+  @Test
+  public void should_fail_if_actual_is_not_a_package_private_class() {
+    assertThatExceptionOfType(AssertionError.class).isThrownBy(() -> classes.assertIsPackagePrivate(someInfo(), Object.class))
+                                                   .withMessage(shouldBePackagePrivate(Object.class).create());
```

Comment:
more a style thingy but we prefer using `AssertionsUtil.expectAssertionError` as shown https://github.com/joel-costigliola/assertj-core/blob/master/CONTRIBUTING.md:
```java
  @Test
  void should_fail_if_optional_is_empty() {
    // GIVEN
    Optional<Object> actual = Optional.empty();
    // WHEN
    AssertionError assertionError = expectAssertionError(() -> assertThat(actual).containsInstanceOf(Object.class));
    // THEN
    then(assertionError).hasMessage(shouldBePresent(actual).create());
  }
```